src/pages/our-principles/index.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react'
+import styled from 'styled-components'
+import { principles } from './_data'
+import Layout from 'components/layout/layout'
+import { Header, Text } from 'components/elements'
+import { SEO, Container, Flex, SectionContainer } from 'components/containers'
+import { localize, WithIntl } from 'components/localization'
+import device from 'themes/device'
+
+type SeparatorInterface = {
+    background: string
+}
+
+const Separator = styled.div<SeparatorInterface>`
+    margin-top: 16px;
+    margin-bottom: 40px;
+    max-width: 1098px;
+    width: 100%;
+    height: 6px;
+    background: ${(props) => (props.background ? props.background : '')};
+`
+
+const ColorHeader = styled(Header)`
+    padding-bottom: 40px;
+    font-size: 48px;
+    color: ${(props) => (props.color ? props.color : '')};
+
+    @media ${device.mobileM} {
+        padding-top: 24px;
+        padding-bottom: 16px;
+        font-size: 28px;
+        line-height: 34px;
+    }
+`
```

Comment:
We can use props and responsive props to achive this without extending `Header`.